.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,13 @@
+contact_links:
+  - name: GTFS.org
```

Comment:
I'd call this Documentation or GTFS Documentation instead of having the URL as a title. 
I think it's a bit more aesthetic, and also more clear: this URL could also be a database of GTFS datasets.

Reply:
Yes this makes sense. I wonder if the main purpose of redirecting to GTFS.org is documentation. I would say it's likely a high percentage of the visit are for things like references pages. I would say this works.